tasks/gitlab_helpers.py

```diff
@@ -281,3 +283,57 @@ def print_entry_points(ctx):
     print(len(entry_points), 'entry points:')
     for entry_point, config in entry_points.items():
         print(f'- {color_message(entry_point, Color.BOLD)} ({len(config)} components)')
+
+
+def compute_gitlab_ci_config_diff(ctx, before: str, after: str):
```

Comment:
I think we should try to keep only task definitions on modules under the `tasks` folder. Can we put this method on a dedicated `lib`?